Bottom: b1f1bba1e2ecfc7229e861b5f03ae29daf71cbf6
Top:    78ae69ceee7423e3c0d9d7f1599e634c4f4ab42c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-05-01 01:59:11 +0100

Refresh of adjust-step-export-to-0-2mm

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 7ec84c0..b81ff61 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -59,7 +59,7 @@
 #endif
 
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(0.2)
 
 static GList *object3d_test_objects = NULL;
